devel/check-container-build-with-prci.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+##
+# It requires the hash to access the RPMS to be stored in the artifact-hash.txt file,
+# or pass on the value as environment variable.
+#
+# Usage:
+#   ARTIFACT_HASH="c8dbfc76-c4eb-11ea-88a6-fa163e90bd6d" ./devel/check-container-build-with-prci.sh
+##
+
+LATEST_FEDORA="fedora-32"
+
+function yield
+{
+    echo "$*" >&2
+} # yield
+
+
+function error-msg
+{
+    yield "ERROR:$*"
+} # error-msg
+
+
+function die
+{
+    local err=$?
+    [ $err -eq 0 ] && err=127
+    error-msg "${FUNCNAME[1]}:$*"
+    exit $err
+} # die
+
+
+function print-build-system
+{
+    local artifact_hash
+    local url
+    local build_system
+
+    artifact_hash="$1"
+    shift 1
+
+    url="http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/${artifact_hash}/rpms/"
```

Comment:
Repo file can be retrieved directly if you know the hash already, it's location is `http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/${artifact_hash}/rpms/freeipa-prci.repo`

Reply:
:+1: Thank you @vashirov 

I have changed it